FindSplash.cmake

```diff
@@ -57,7 +57,7 @@
 # Required cmake version
 ###############################################################################
 
-cmake_minimum_required(VERSION 2.8.5)
+cmake_minimum_required(VERSION 2.8.12.2)
```

Comment:
FPHSA macro introduces the `FOUND_VAR` option in [CMake 2.8.11+](http://www.kitware.com/blog/home/post/492) ([previous comment in mallocMC](https://github.com/ComputationalRadiationPhysics/cmake-modules/pull/8#issuecomment-125618929))